definitions/divorce/json/CaseField/CaseField-share-a-case.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
file name: `CaseField-share-a-case-nonprod.json`

you've missed: -nonprod

Reply:
Opps!!!! Yes. It wasn't me 🤥

Reply:
removed